test/animals.js

```diff
@@ -1,5 +1,5 @@
 var expect = require('chai').expect;
-var ObjectTemplate = require('supertype');
+var ObjectTemplate = require('supertype').default;
```

Comment:
why is `.default` needed?

Reply:
I wanted to be explicit with the imports, also it causes an error at runtime if I don't specifically do default.
ObjectTemplate is exported like so in Supertype (the following is compiled javascript in index.js)

```
ObjectTemplate_1.ObjectTemplate.init();
exports.default = ObjectTemplate_1.ObjectTemplate;
```

Reply:
hmmm - we may need to re-visit how we're exporting then - consumers should be able to consume supertype via simply `require('supertype')`

Reply:
I don't think that's possible. Besides, this is only an issue in JS. In Typescript you have to explicitly specify everything you want.

Reply:
Also, our module resolution for supertype before followed no proper conventions, there was something similar to UMD, but not even that standard. Now we are just doing commonjs.

Also I think this is an issue because you cannot `require('supertype')` and also export other things as well like `Supertype` or `ObjectTemplate` or anything else. You only get to have one object/function to export. Even still I'm not sure the `export` keyword in Typescript compiles to something that's functionally equivalent to how we are currently exporting stuff in Supertype.